pkg/authn/helper_test.go

```diff
@@ -136,3 +149,61 @@ func TestBadOutput(t *testing.T) {
 		t.Errorf("Authorization() = %v", got)
 	}
 }
+
+// TestHTTPSURL checks that helper saving https works
+func TestHTTPSURL(t *testing.T) {
+	output := `{"ServerURL":"","Username":"mainuser","Secret":"FVLGCQr_a-FZJM2ON227-YXdAEJn3oZZnnWs6Jd6iVE"}`
```

Comment:
Is `ServerURL` needed? Does it not default to `""` if unset?

Reply:
It is not needed. It is what _docker-credential-secretservice_ returns. And I want to ensure, that it does not cause any problems.